web-client/src/models/offers/index.ts

```diff
@@ -45,17 +49,21 @@ export class Offer implements IOffer {
       firebase.firestore.DocumentData
     >,
     cavUserSnapshot: User,
+    requestSnapshot: Request | null,
     message: string,
     status: OfferStatus,
     createdAt = firestore.Timestamp.now(),
+    seenAt = firestore.Timestamp.now(),
```

Comment:
same as I mentioned in the models in the functions

Reply:
fixed